src/azure-cli/azure/cli/command_modules/acr/connected_acr.py

```diff
@@ -0,0 +1,298 @@
+                'ConnectedRegistryUpdateParameters', 'SyncProperties', 'LoggingProperties')
+    sync_properties = SyncProperties(
+        token_id=sync_token_id,
+        schedule=sync_schedule,
+        message_ttl=sync_message_ttl,
+        sync_window=sync_window
+    )
+    logging_properties = LoggingProperties(
+        log_level=log_level,
+        audit_log_status=sync_audit_logs_enabled
+    )
+    connected_acr_update_parameters = ConnectedRegistryUpdateParameters(
+        sync_properties=sync_properties,
+        logging=logging_properties,
+        client_token_ids=client_token_ids
+    )
+
+    # Add or remove the repo permissions from the sync token scope map id
+    if add_repository or remove_repository or mode:
+        mode = mode if mode else current_mode
+        _ = _update_sync_token_scope_map(cmd, resource_group_name, registry_name, connected_acr_name,
+                                         sync_token_id, add_repository, remove_repository, mode)
+    try:
+        return client.update(resource_group_name=resource_group_name,
+                             registry_name=registry_name,
+                             connected_registry_name=connected_acr_name,
+                             connected_registry_update_parameters=connected_acr_update_parameters)
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def acr_connected_acr_delete(cmd,
+                             client,
+                             connected_acr_name,
+                             registry_name,
+                             yes=False,
+                             resource_group_name=None):
+
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+
+    #TODO disable tokens
+    user_confirmation("Are you sure you want to delete the Connected Registry '{}' from '{}'?".format(
+        connected_acr_name, registry_name), yes)
+    try:
+        return client.delete(resource_group_name, registry_name, connected_acr_name)
+    except ValidationError as e:
+        raise CLIError(e)
+
+
+def acr_connected_acr_list(cmd,
+                           client,
+                           registry_name,
+                           parent=None,
+                           cascading=False,
+                           resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+    #TODO Cascading logic and different formattings.
+    return client.list(resource_group_name, registry_name)
+
+
+def acr_connected_acr_show(cmd,
+                           client,
+                           connected_acr_name,
+                           registry_name,
+                           resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name)
+    return client.get(resource_group_name, registry_name, connected_acr_name)
+
```

Comment:
@rosanch we also need a command for the deactivate API. See https://msazure.visualstudio.com/One/_git/DevServices-ContainerRegistry-Service/pullrequest/3733243